src/components/garagePromotionCard.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, MouseEvent } from "react"
+
+import UploadImagesIcon from "./icons/uploadImage"
+import ArrowRightTeal from "./icons/arrowRight"
+import { FixedCard } from "./card/fixed"
+
+interface Props {
+  dealerName: string
+  dealerLocation: {
+    address?: string
+    zipCode: string
+    city?: string
+  }
+  title?: string
+  previewLabel: string
+  mainImage?: string
+  logo?: string
+  linkToDealerPage?: string
+  handleOnClick?: (event: MouseEvent<HTMLButtonElement>) => void
+  handleOnError?: (event: React.ChangeEvent<HTMLInputElement>) => void
+  handleOnClickText?: (event: MouseEvent<HTMLButtonElement>) => void
+}
+
+const GaragePromotionCard: FC<Props> = ({
+  dealerName,
+  dealerLocation,
+  title,
+  previewLabel,
+  mainImage,
+  logo,
+  linkToDealerPage,
+  handleOnClick,
+  handleOnError,
+  handleOnClickText,
+}) => {
+  return (
+    <FixedCard
+      renderImage={() => (
+        <div>
+          {!mainImage.includes("null") ? (
+            <img
+              src={mainImage}
+              className="promotionImage w-12/12 object-cover"
+              alt={title}
+              onClick={() => {
+                handleOnClick
+              }}
+              onError={() => {
+                handleOnError
+              }}
+            />
+          ) : (
+            <UploadImagesIcon width="72px" height="72px" />
```

Comment:
displaying an upload image is a DH-specific requirement. can this be just passed as the main image instead of adding logic?

I like the "principle of least surprise" and looking at the props, it would definitely surprise me that there was some implicit image being displayed when I pass none